xcodeproj/internal/runner.template.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -euo pipefail
+
+readonly bazelrc="$PWD/%bazelrc%"
+
+cd "$BUILD_WORKSPACE_DIRECTORY"
+
+bazelrcs=("--bazelrc=$bazelrc")
+if [[ -s ".bazelrc" ]]; then
+  bazelrcs+=("--bazelrc=.bazelrc")
+fi
+
+echo 'Generating "%project_name%.xcodeproj"'
+
+"%bazel_path%" \
```

Comment:
Maybe this isn't being substituted correctly.

> swiftlint_xcodeproj-runner.sh: line 17: bazel: command not found

Reply:
Should it be something other than `bazel` for you?

Reply:
Yes I specify the bazel path. I can debug more.

Reply:
It would be ideal if this path could be controlled somewhere

Reply:
That's the point here, it should be the value used in `xcodeproj.bazel_path`. I'll also debug as well.

Reply:
I think `bazel_path` needs to be set here: https://github.com/buildbuddy-io/rules_xcodeproj/blob/af31065b421be7d1bf4517a3ab2848ef9ca3fa8f/xcodeproj/internal/xcodeproj_macro.bzl#L90-L95

If I hardcode it there to the absolute path for my custom bazel path it works.

Reply:
Yep, that's it. I'll get that in when I push my other changes.